config/database.yml.txt

```diff
@@ -0,0 +1,30 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: postgresql
+  pool: 5
+  timeout: 5000
+
+development:
+  adapter: postgresql
+  encoding: unicode
+  database:
```

Comment:
I suggest you to use some default name for a development database, for example, `supersonic_development` to simplify local deployment.